pkg/version/version.go

```diff
@@ -0,0 +1,41 @@
+package version
+
+import (
+	"github.com/openshift/cluster-samples-operator/pkg/apis/samplesoperator/v1alpha1"
+	"k8s.io/apimachinery/pkg/version"
+)
+
+var (
+	// commitFromGit is a constant representing the source version that
+	// generated this build. It should be set during build via -ldflags.
+	commitFromGit string
+	// versionFromGit is a constant representing the version tag that
+	// generated this build. It should be set during build via -ldflags.
+	versionFromGit string
+	// major version
+	majorFromGit string
+	// minor version
+	minorFromGit string
+)
+
+// Get returns the overall codebase version. It's for detecting
+// what code a binary was built from.
+func Get() version.Info {
+	return version.Info{
+		Major:      majorFromGit,
+		Minor:      minorFromGit,
```

Comment:
add patch

Reply:
so yeah not inclined to add patch ... we can revisit if/when @smarterclayton chimes in and when you get back from PTO

In conjunction with not finding it in this space in openshift/origin, would have to research how to compile that information